backend/src/test/resources/application.properties

```diff
@@ -1,4 +1,4 @@
 spring.data.mongodb.database=eventBudgetTestDB
 spring.mongodb.embedded.version=3.5.5
 
-neuefische.capstone-event-budget.jwt.secret=testsecretwasgehtabyo
\ No newline at end of file
+neuefische.capstone-event-budget.jwt.secret="jwt-secret-for-test-purposes-not-environment-variable-otherwise-single-tests-crash-;-)"
```

Comment:
I am not sure, but this should not visible in github.

Reply:
Das ist in der config für die Tests und daher ein bisschen hakelig als Umgebungsvariable zu setzen. Im normalen Source Code habe ich das aber getan.